test/e2e/framework/rc_util.go

```diff
@@ -36,12 +36,13 @@ import (
 )
 
 // RcByNamePort returns a ReplicationController with specified name and port
-func RcByNamePort(name string, replicas int32, image string, port int, protocol v1.Protocol,
+func RcByNamePort(name string, replicas int32, image string, containerArgs []string, port int, protocol v1.Protocol,
 	labels map[string]string, gracePeriod *int64) *v1.ReplicationController {
 
 	return RcByNameContainer(name, replicas, image, labels, v1.Container{
 		Name:  name,
 		Image: image,
+		Args:  containerArgs,
```

Comment:
I'm fine with this change, but you are passing the same argument multiple times, when not just make it the default at this callsite?

Reply:
/hold

(cause there's already a lgtm)

Reply:
Original thinking was: I'm not sure if this is being called/used outside of this, so best to leave a "blank" default.

I'll de-dupe the args now since y'all definitely know better than me 😎